pkg/config/sink.go

```diff
@@ -221,7 +221,8 @@ func (c *CSVConfig) validateAndAdjust() error {
 		}
 	default:
 		return cerror.WrapError(cerror.ErrSinkInvalidConfig,
-			errors.New("csv config delimiter contains more than one character"))
+			errors.New("csv config delimiter contains more than one character, note that escape "+
```

Comment:
I think you can use `GenWithStack`.

Reply:
This function is called internally by WrapError, but WrapError is safer in the case of nil errors.